src/main/scala/akka/persistence/cassandra/query/scaladsl/CassandraReadJournal.scala

```diff
@@ -106,9 +106,12 @@ class CassandraReadJournal(system: ExtendedActorSystem, config: Config)
 
   @volatile private var sessionUsed = false
 
+  private val metricsCategory = s"cassandra.readjournal.${system.name}.hash.${System.identityHashCode(this)}"
```

Comment:
Ok, append value of an AtomicInteger counter or hash code, but not for the first value, then it will be compatible with the new solution when only one plugin is used, i.e. "cassandra-query-journal"